app/controllers/students/base_controller.rb

```diff
@@ -8,7 +8,7 @@ def must_be_student
   end
 
   def current_team
-    @current_team ||= current_user.teams.first
+    @current_team ||= current_user.teams.last
```

Comment:
This sounds like a much better approach definitely prevent future issue 
So we can change it to this 
`current_user.teams.in_current_season`
?